clang/lib/CIR/CodeGen/CIRGenModule.cpp

```diff
@@ -3193,6 +3193,39 @@ void CIRGenModule::addReplacement(StringRef Name, mlir::Operation *Op) {
   Replacements[Name] = Op;
 }
 
+void CIRGenModule::replacePointerTypeArgs(cir::FuncOp OldF, cir::FuncOp NewF) {
+  auto optionalUseRange = OldF.getSymbolUses(theModule);
+  if (!optionalUseRange)
+    return;
+
+  for (auto U : *optionalUseRange) {
+    // FIXME: Handling TryCallOp here once they have the same base op.
+    auto Call = mlir::dyn_cast<cir::CallOp>(U.getUser());
```

Comment:
Oversight on my side, TryCallOp only shows up in FlattenCFG, so feel free to remove the comment.

Reply:
Yeah, it looks better to me too